docs/INSTALLATION.md

```diff
@@ -178,6 +178,10 @@ Daarnaast zijn er bronnen die binnen KISS doorzocht moeten worden.
 | `INTERNE_TAAK_TOKEN`                    | Token voor de Objecten API voor Interne Taken  <br/> **(niet te gebruiken met `INTERNE_TAAK_CLIENT_SECRET` en `INTERNE_TAAK_CLIENT_ID`)** <details> <summary>Meer informatie </summary>In de meeste gevallen identificeert KISS zich bij een Objectenregistratie m.b.v. een `TOKEN`. In sommige gevallen is het nodig om de authenticatie in de Objecten API, voor Afdelingen, Groepen en Interne Taken, Medewerkers in te regelen m.b.v. een `client secret` en een `client id`. Dit is bv. het geval als je KISS gebruikt i.c.m. de e-Suite. Afhankelijk van de situatie moet je dus een Token inregelen, en in andere gevallen een id+secret. **NOOIT ALLEBEI!** </details> |
 | `INTERNE_TAAK_CLIENT_SECRET`            | Client Secret voor de Interne Taken API <br/> **(niet te gebruiken in combinatie met `INTERNE_TAAK_TOKEN`)**                      |
 | `INTERNE_TAAK_CLIENT_ID`                | Client ID voor de Interne Taken API <br/> **(niet te gebruiken in combinatie met `INTERNE_TAAK_TOKEN`)**                          |
+| `VAC_OBJECTEN_BASE_URL`                 | URL van de Objecten API voor Vraag Antwoord Combinaties <details> <summary>Meer informatie</summary> Bijvoorbeeld: `https://objectenregister.mijngemeente.nl` </details>            |
+| `VAC_OBJECT_TYPE_URL`          | URL van het Objecttype Vac <details> <summary>Meer informatie</summary> Bijvoorbeeld `https://objecttypenregister.mijngemeente.nl/api/v2/objecttypes/f83fdc48-5ddb-4b1a-a347-e20092031399` </details> |
+| `VAC_OBJECT_TYPE_VERSION`             | Versienummer van het Objecttype Vac <details> <summary>Meer informatie</summary> Bijvoorbeeld `2` <br /> KISS schrijft Vacs in het Objectenregister. Hierbij moet je altijd de versie van het objecttype meegeven. Omdat het per gemeente kan verschillen welke versie de meest recente is, moet je hier invullen welk versienummer KISS moet meegeven. </details>                                                          |

```

Comment:
@nijmra , ik zie dat hier is toegevoegd: `VAC_OBJECT_TYPE_VERSION`
In hoeverre moet de Sync Tool daar niet ook naar luisteren?